common/changes/@microsoft/webpack5-loader-load-themed-styles/issue-3863_2023-01-06-00-24.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/webpack5-loader-load-themed-styles",
```

Comment:
This name is awkward. Since we're starting anew here, should we call this `webpack5-load-themed-styles-loader`?

Reply:
Changed to this